data/political/2022-2032-house-senate-congressional-districts/index.html

```diff
@@ -143,3 +151,22 @@ <h3 id="UnitedStatesCongressionalDistricts">United States Congressional District
     %}
   </div>
 </div>
+
+<div class="grid package">
+  <div class="grid__col grid__col--12-of-12">
+    <h3 id="CombinedDistricts">Combined Political Districts 2022 to 2032</h3>
+    <div class="grid__col grid__col--12-of-12 package-content">
+      <img class="inline-text-left outline" src="{% link images/HouseSmall2.png %}" alt="StateHouseNew"
+        loading="lazy" />
+      <p>Utah District Combination Areas 2022 is comprised of all 4 politcal districts including the House, Senate, U.S. Congressional, and School Board districs.</p>
```

Comment:
```suggestion
      <p>Utah District Combination Areas 2022 to 2032 is comprised of all 4 political districts including the House, Senate, U.S. Congressional, and School Board districts.</p>
```

Reply:
What references do you want changed on the 2012 page?

Why would we make this suggested change? The name of the layer is _Utah District Combination Areas 2022_. I did not create it. It probably should have be created with the new naming convention but the cat's out of the bag now.